api/lambdas/sqs-to-converter/index.js

```diff
@@ -402,6 +418,24 @@ function addMissingRequests(conversion) {
   });
 }
 
+async function notifyApi(params, log) {
+  if (params.cxId && params.patientId) {
+    let attempt = 0;
+    while (attempt++ < MAX_API_NOTIFICATION_ATTEMPTS) {
+      try {
+        log(`(${attempt}) Notifying API w/ ${JSON.stringify(params)}`);
+        await ossApi.post(docProgressURL, null, { params });
+      } catch (error) {
+        const msg = "Error notifying API, trying again";
+        const extra = { url: docProgressURL, statusCode: error.response?.status, attempt, error };
+        log(msg, extra);
+        Sentry.message(msg, { extra, level: "info" });
```

Comment:
NBD, but should we have some exponential back-off here? 

ie if we send 5 reqs in 100ms, it might have a different outcome if we send the second after 500ms, then 1s, then 2s etc.

Reply:
done